portal-web/docroot/WEB-INF/tiles-defs.xml

```diff
@@ -76,6 +76,11 @@
 		<put name="content" value="/portal/update_password.jsp" />
 	</definition>
 
+	<definition extends="portal" name="portal.update_password_up">
+		<put name="title" value="new-password" />
+		<put name="content" value="/portal/forgot_password.jsp" />
```

Comment:
Could we rename the `forgot_password.jsp` to `update_password_utility_page.jsp`?